me.sergiotarxz.Exd.yml

```diff
@@ -0,0 +1,120 @@
+id: "me.sergiotarxz.Exd"
+runtime: "org.gnome.Platform"
+runtime-version: "47"
+sdk: "org.gnome.Sdk"
+finish-args:
+  - --share=ipc
+  - --device=all
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --allow=bluetooth
+  - --socket=system-bus
```

Comment:
system-bus is not allowed why do you need it?

Reply:
> A IDE to create tickets and stickers for thermal printers.

why does this need system-bus?

Reply:
Because it has to talk with low energy bluetooth printers using dbus bluez.

Reply:
Without this I will have to drop support for cat printers.

Reply:
there is a dbus name for bluetooth. it's even documented.

Reply:
Unless you can tell me a way to use dbus bluez without requiring the system bus, I did not find that in the docs.

Reply:
> there is a dbus name for bluetooth. it's even documented.

![image](https://github.com/user-attachments/assets/10d57a00-db23-4d47-a743-d7cde4297c42)

I cannot find it

Reply:
I have tried using --talk-name=org.bluez but system socket is not visible in the app and the session socket doesn't have bluez anyway.

Reply:
I solved it with --system-talk-name=org.bluez, thank you.